src/views/Item/ItemOKRs.vue

```diff
@@ -122,6 +112,7 @@ export default {
       'keyResults',
       'objectives',
       'periods',
```

Comment:
Dette har muligens vært et problem fra før: Hvis det ikke eksisterer noen perioder, og man velger admin-fanen, lager en ny periode, går tilbake til "OKRs", får man fortsatt beskjed om at det ikke eksisterer perioder. Den virker ellers reaktiv for nye/endrede perioder om det finnes fra før.